<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_row">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Row Size : </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_row">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_col">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Col Size : </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_col">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_array" native="true">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_create">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_quit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
